package.json

```diff
@@ -49,6 +49,7 @@
     "fb": "^2.0.0",
     "helmet": "^3.21.1",
     "i18n": "^0.8.3",
+    "minimist": "^1.2.5",
```

Comment:
I think we don't need this one 🙂 
The problem with audit lies within the dependencies of the dependencies.
So, what needs to be done is `npm outdated` and then look and update outdated dependencies, some of which in their turn rely on `minimist`. 🙂

Reply:
@pheeria I installed the latest versions of `jest` and `nodemon` and updated other packages to their latest wanted version. Everything seems to work for me and `npm audit` does not detect any vulnerabilities.